src/krishna/Shop.java

```diff
@@ -0,0 +1,89 @@
+package krishna;
+
+public class Shop {
+	String maggie="Maggie",dosa= "Dosa",pouches = "Pouches",panipuri = "Panipuri",masala= "Masala";
+	int maggieQuan=50,dosaQuan=43,pouchesQuan=39,panipuriQuan=43,masalaQuan=73;
+	
+	void setInitial(int maggieQ,int dosaQ,int pouchesQ,int panipuriQ,int masalaQ) {
+		if(maggieQ<=50) {
+			maggieQuan = maggieQuan-maggieQ;
+			System.out.println("Available stock "+maggie+": " +maggieQuan);
+		}else{
+			maggieQuan=0;
```

Comment:
If request stock is not available mean we are setting direct it zero, 
what if second time your calling same method and let say you request 20 packs, then this looks not logically correct